### ISSUE TITLE
AWT failure - NO VASSAL CODE ON STACK - AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge

### ISSUE THREAD

**Issue by uckelman** [vassalengine/vassal#2975]
| Bugzilla |  |
|----------|----------|
| ID | 13170 |
| Reported | 2020-07-13 04:03:00 +0200 |
| Modified | 2020-10-27 17:16:32 +0100 |
| Product | VASSAL |
| Component | External |
| Version | 3.3.1 |
| Platform | All |
| OS | All |
| Status | RESOLVED |
| Resolution | WORKSFORME |
| Priority | unspecified |
| Severity | normal |
| Milestone | --- |
| Reporter | [Joel Uckelman](mailto:uckelman@nomic.net) |
| Assignee | [Bugs](mailto:bugs@vassalengine.org) |
| CC | brian@brianreynolds.net |

| Attachments | |
|-------|-------|
| [errorLog](http://www.vassalengine.org/files/bz/12196) | [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-13 04:03:00 +0200 text/plain 2349 the errorLog |

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-13 04:03:52 +0200
<pre>Trying to start the program</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2020-07-13 04:03:53 +0200
<pre>Created attachment 12196
the errorLog</pre>
